File: openbis/source/java/ch/systemsx/cisd/openbis/public/resources/api/v3/as/dto/event/EntityType.js
/**
 * @author pkupczyk
 */
define([ "stjs", "as/dto/common/Enum" ], function(stjs, Enum) {
	var EntityType = function() {
		Enum.call(this, [  "ATTACHMENT", "DATA_SET", "EXPERIMENT", "SPACE", "MATERIAL", "PROJECT", "PROPERTY_TYPE", "SAMPLE",
                              "VOCABULARY", "AUTHORIZATION_GROUP", "TAG" ]);
	};
	stjs.extend(EntityType, Enum, [ Enum ], function(constructor, prototype) {
	}, {});
	return new EntityType();
})

File: openbis_api/source/java/ch/ethz/sis/openbis/generic/asapi/v3/dto/event/EntityType.java
package ch.ethz.sis.openbis.generic.asapi.v3.dto.event;

import ch.systemsx.cisd.base.annotation.JsonObject;

@JsonObject("as.dto.event.EntityType")
public enum EntityType
{
    ATTACHMENT, DATA_SET, EXPERIMENT, SPACE, MATERIAL, PROJECT, PROPERTY_TYPE, SAMPLE,
    VOCABULARY, AUTHORIZATION_GROUP, TAG;
}
